inlong-sort/sort-connectors/kafka/src/main/java/org/apache/inlong/sort/kafka/table/KafkaDynamicTableFactory.java

```diff
@@ -289,9 +336,11 @@ public DynamicTableSink createDynamicTableSink(Context context) {
         final EncodingFormat<SerializationSchema<RowData>> valueEncodingFormat =
                 getValueEncodingFormat(helper);
 
+        final String innerValueDecodingFormat = getInnerDecodingFormat(helper);
```

Comment:
Once here innerValueDecodingFormat  is used for raw datatype , validate physicalDataType?

Reply:
Good idea.